src/EA.Weee.Web/ViewModels/Shared/EvidenceNoteRowViewModel.cs

```diff
@@ -77,6 +77,8 @@ public string SchemeViewRouteName
                         return SchemeTransferEvidenceRedirect.ViewRejectedTransferEvidenceRouteName;
                     case NoteStatus.Returned:
                         return SchemeTransferEvidenceRedirect.ViewReturnedTransferEvidenceRouteName;
+                    case NoteStatus.Void:
```

Comment:
Update the related unit test for this new status and route name.

Reply:
done